src/Config/FileSystemRuntimeConfigLoader.cs

```diff
@@ -58,6 +58,11 @@ public FileSystemRuntimeConfigLoader(IFileSystem fileSystem, string baseConfigFi
         _fileSystem = fileSystem;
         _baseConfigFileName = baseConfigFileName;
         ConfigFileName = GetFileNameForEnvironment(Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT"), false);
+        // if fileName based on environment is empty, then use the default config file name.
```

Comment:
The function: `GetFileNameForEnvironment` should return the default config file name if the fileName based on environment is empty - the 3rd index is string.Empty in the environment variable loop. Did you observe this is not happening?

Reply:
yes, it was returning empty when the config file is not found, as can be seen in the Logs. Due to which logs are like this `Unable to find config File: does not exist.`